src/components/Toast/__stories__/toast.stories.mdx

```diff
@@ -248,7 +248,7 @@ After a user has done an action, provide feedback to close the loop. For example
             type={Toast.types.POSITIVE}
             actions={actions}
             onClose={onCloseCallback}
-            autoHideDuration={5000}
+            autoHideDuration={50000000}
```

Comment:
I guess this has been changed for test purposes - fix needed

Reply:
sure thanks.